usaspending_api/etl/elasticsearch_loader_helpers/transform_data.py

```diff
@@ -137,6 +136,10 @@ def transform_covid19_faba_data(worker: TaskSpec, records: List[dict]) -> List[d
     return list(results.values())  # don't need the dict key, return a list of the dict values
 
 
+def transform_recipient_profile_data(worker: TaskSpec, records: List[dict]) -> List[dict]:
```

Comment:
The elasticsearch indexer command config expects a `data_transform_func` value that points to a function to run to transform the records. The management command expects this to be a valid function, so I just added a basic function that just returns the records since we don't need to transform them.

```python
config = {
            "base_table": "recipient_profile",
            "base_table_id": "id",
            "create_award_type_aliases": False,
            "data_transform_func": transform_recipient_profile_data,
        }
```

Reply:
Is there a better way to go about this besides defining a obsolete method?

Reply:
Perhaps you could take advantage of `lambda` in the config json.

Reply:
Another idea is to update the django command to allow None (or something besides a shell of a function).

Reply:
I updated the management command to support a value of `None` whenever the `data_transform_func` value isn't going to be needed.